crates/aptos/Cargo.toml

```diff
@@ -62,6 +62,7 @@ diesel-async = { workspace = true }
 dirs = { workspace = true }
 futures = { workspace = true }
 hex = { workspace = true }
+indoc = { workspace = true }
```

Comment:
Can we do this without adding `indoc` as a dependency, or is it necessary?

Reply:
I mean.... its pretty nice to have and its a very simple dependency, i find its generally useful and probably already included transitively